todo_app/app.py

```diff
@@ -75,11 +75,9 @@ def delete_todo(id):
         trello = Trello()
         trello.delete_item(id)
         return redirect('/')
-    
     return app
 
 file = __file__
-
-#if __name__ == 'todo_app.app':
-    #create_app()
+if __name__ == 'todo_app.app':
+    create_app()
```

Comment:
Do you ever run the file this way? This should be unnecessary for the Gunicorn/flask commands, in which case I tend to recommend skipping this so that you only have to support 2 ways to run the app, rather than allowing a 3rd that you have to maintain!